rapids-cmake/cpm/versions.json

```diff
@@ -39,6 +39,11 @@
       "version" : "1.7.0",
       "git_url" : "https://github.com/NVIDIA/libcudacxx.git",
       "git_tag" : "${version}"
+    },
+    "cuco" : {
+      "version" : "0.0.1",
+      "git_url" : "https://github.com/NVIDIA/cuCollections.git",
+      "git_tag" : "55029034c3f82bca36148c9be29941b37492394d"
```

Comment:
Is there a reason we don't use shallow git clones for packages in `versions.json`?

Reply:
Clones are shallow by default, unless versions.json specifies otherwise.

Reply:
Awesome. I looked briefly but didn't see where that was defaulted. I had some issues with CPM timing out earlier this week when my internet was being flaky. I assumed it was due to full clones.

Reply:
The default behavior for shallow vs non is here: https://docs.rapids.ai/api/rapids-cmake/nightly/packages/rapids_cpm_versions.html#cpm-versions

Reply:
> If no such field exists the default is false.

That conflicts with @vyasr's statement above that clones are shallow (true) by default. 🤔

Reply:
The docs are lying, it's true by default. I'll fix this. https://github.com/rapidsai/rapids-cmake/blob/464362f90b122b8a112e1675626fd251676544bb/rapids-cmake/cpm/detail/package_details.cmake#L62-L64